lib/datadog/statsd/message_buffer.rb

```diff
@@ -92,6 +123,33 @@ def ensure_sendable!(message_size)
       def bytesize_threshold
         @bytesize_threshold ||= (max_payload_size - PAYLOAD_SIZE_TOLERANCE * max_payload_size).to_i
       end
+
+      def create_flush_thread
+        flush_thread = Thread.new(&method(:flush_loop))
+        flush_thread.name = 'Statsd MessageBuffer' if flush_thread.respond_to?(:name=)
```

Comment:
For better or for worse, the way we've done this in other files is e.g.,
```ruby
@sender_thread.name = "Statsd Sender" unless Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.3')
```
so let's stick to that pattern.

Reply:
I've changed the code to follow the same way in the commit https://github.com/DataDog/dogstatsd-ruby/pull/154/commits/f5f8686b1161aced41df61e95a4744559cfccab2.